Dockerfile

```
@@ -7,7 +7,8 @@ WORKDIR /app
 # COPY Gemfile /app/Gemfile
 # COPY Gemfile.lock /app/Gemfile.lock
 COPY ./project /app
-RUN bundle install
+RUN gem update bundle
+RUN bundle install --jobs=10
```

@daiphone0228 
[imo]

```suggestion
RUN bundle install --jobs=`getconf _NPROCESSORS_ONLN`
```

[【Rails】bundle install \-\-jobs（\-j）に指定するジョブ数を動的に設定するため、最大値について調べた \- スリ飯屋MaLankaのフリーエンジニアな日々](https://www.malanka.tech/entry/dynamically-set-number-of-jobs-for-bundle-install)

---

ご提案ありがとうございます。
動的に定義できる方がいいと思うので採用させていただきます。
修正しました。